release-please-config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "bootstrap-sha": "640550975a412d71fc35875103e1c833237b168d",
```

Comment:
`sha` of the 0.10.0 release commit. This will be ignored once the first release PR is merged, so we can delete this later.